ntoskrnl/mm/ARM3/procsup.c

```diff
@@ -997,7 +966,13 @@ MmInitializeProcessAddressSpace(IN PEPROCESS Process,
     MiInitializePfn(PageFrameNumber, PointerPte, TRUE);
 
     /* Do the same for hyperspace */
-    PointerPde = MiAddressToPde((PVOID)HYPER_SPACE);
```

Comment:
Lol, how did it work before for 64 bits?

Reply:
This actually breaks it. This is a leftover from a bad git rebase.